backends/redis.go

```diff
@@ -36,41 +60,40 @@ func NewRedisBackend(cfg config.Redis) *Redis {
 		}
 	}
 
-	client := redis.NewClient(options)
+	redisClient := RedisDBClient{redis.NewClient(options)}
 
-	_, err := client.Ping().Result()
+	_, err := redisClient.client.Ping().Result()
 
 	if err != nil {
 		log.Fatalf("Error creating Redis backend: %v", err)
 	}
 
 	log.Infof("Connected to Redis at %s:%d", cfg.Host, cfg.Port)
 
-	return &Redis{
+	return &RedisBackend{
 		cfg:    cfg,
-		client: client,
+		client: redisClient,
 	}
 }
 
-func (redis *Redis) Get(ctx context.Context, key string) (string, error) {
-	res, err := redis.client.Get(key).Result()
+func (back *RedisBackend) Get(ctx context.Context, key string) (string, error) {
```

Comment:
Nitpick: `back` is a confusing name to me. Consider renaming to simply `b`.

Reply:
Good catch. Done